delivery_dropoff_site/models/res_partner.py

```diff
@@ -0,0 +1,51 @@
+# coding: utf-8
+# Copyright (C) 2014 - Today: Akretion (http://www.akretion.com)
+# Copyright (C) 2018 - Today: GRAP (http://www.grap.coop)
+# @author Aymeric Lecomte <aymeric.lecomte@akretion.com>
+# @author David BEAL <david.beal@akretion.com>
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment:
Author is already defined in contributor's readme file. Dont you think so it should be removed from all py files ?

Reply:
Well, 
- I dont see any OCA rules that says that we should remove authors and contributors from the header section. 
- There are such header in most of the OCA modules.

I think it's not the place here to talk about new general OCA rules. If you think it's usefull, please make an issue on maintainer-tools.

In the meantime, let's go forward with that PR.

thanks for your review.

Reply:
@MeetKD, could you update your review ?
thanks !